content/docs/atom-guide/look-dev/materials/create-standardpbr-material.md

```diff
@@ -13,30 +13,30 @@ This tutorial covers the following concepts:
 - Using texture file masks
 
 ## Before starting
-A material can have multiple textures, such as base color, normal map, and roughness. These textures contain different data that are combined to create the overall look of the material. You can design the textures for a material using external tools, such as [Substance Designer](https://www.substance3d.com/products/substance-designer/) and [Materialize](http://boundingboxsoftware.com/materialize/). In this tutorial, we use ready-made materials from [Texture Haven](https://texturehaven.com/). 
+A material can have multiple textures, such as base color, normal map, and roughness. These textures contain different data that are combined to create the overall look of the material. You can design the textures for a material using external tools, such as [Substance Designer](https://www.adobe.com/products/substance3d-designer.html) and [Materialize](http://boundingboxsoftware.com/materialize/). In this tutorial, we use ready-made materials from [Poly Haven](https://polyhaven.com/). 
 
 ### Download texture files
-Download the [castle brick material](https://texturehaven.com/tex/?t=castle_brick_02_red) in any resolution from Texture Haven. When downloading, make sure to choose **All Maps** to download all textures that make up this material.
+Download the [castle brick material](https://polyhaven.com/a/castle_brick_02_red) in any resolution from Poly Haven. To download all the required maps, click the hamburger button next to the **Download** button, then click buttons with any of the file extensions next to AO, Diffuse, Displacement, Normal, and Rough.
 
-After downloading, move and extract the zip file to the folder `<project-folder>/Materials/TextureHaven/`.
+After downloading, move and extract the zip file to the folder `<project-folder>/Materials/PolyHaven/`.
```

Comment:
```suggestion
After downloading, move the maps to the folder `<project-folder>/Materials/PolyHaven/`.
```
I think the new process requires changing this step as well.  Feel free to rework my suggestions if you think users should save to `<project-folder>/Materials/PolyHaven/` directly.

Reply:
I must have overlooked the step with zip extraction, thanks for catching and mentioning it!